### PR TITLE
docs: Update production checklist after changes to the roles mgmt

### PR DESCRIPTION
**Pull request by keydunov**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Deployment/Production-Checklist.md [hunk 1]
```diff
@@ -25,6 +25,37 @@ lead to security vulnerabilities. You can read more on the differences between
 CUBEJS_DEV_MODE=false
 ```
 
+## Set up Refresh Worker
+
+To refresh in-memory cache and [pre-aggregations][ref-pre-aggregations] in the background, we recommend running
+a separate Cube.js refresh worker instance. This allows your Cube.js API
```
Comment (hassankhan) [inline docs/content/Deployment/Production-Checklist.md:31]:
```suggestion
a separate Cube.js Refresh Worker instance. This allows your Cube.js API
```

File: docs/content/Deployment/Production-Checklist.md [hunk 2]
```diff
@@ -83,71 +114,7 @@ If you want to run Cube.js in production without Redis, you can use
 | to manage the query queue.
 <!-- prettier-ignore-end -->
 
-## Set up Pre-aggregations Storage
```
Comment (hassankhan) [inline docs/content/Deployment/Production-Checklist.md:86]: We should probably do a quick sweep to make sure the headings removed below don't have any links pointing to them

**Review by hassankhan — APPROVED** [review]
PR looks great 😀

File: docs/content/Deployment/Production-Checklist.md (hunk 2 above)
Comment (hassankhan) [reply docs/content/Deployment/Production-Checklist.md:86]: Just did a sweep, doesn't seem like anything is broken (in the docs, at least)

**Comment by keydunov** [comment]
Thank you @hassankhan!

**Pull request merged**